views/trials-details.html

```diff
@@ -191,29 +191,18 @@ <h2 class="disabled">{{ name }}</h2>
     <h2>Identifiers</h2>
     <dl class="id-list">
       <dt>
-        OpenTrials ID:
+        OpenTrials
       </dt>
       <dd>
         {{ trial.id }}
       </dd>
 
-      {% if primaryId %}
+      {% for identifier in trial.identifiers %}
       <dt>
-        Primary ID:
+        {{ identifier.name }}
```

Comment:
The names should be things like "ClinicalTrials.gov", "EU CTR", etc. The name should be corrected in the DB-level.